docs/dcp2_operating_procedures.rst

```diff
@@ -0,0 +1,253 @@
+consortium approve of a schema change **before** it 
+
+- affects the master branch of the `metadata-schema`_ repository,
+
+- is published on `schema.humancellatlas.org`_, and
+
+- is used by metadata staged for import into the production instance of TDR
+
+.. _schema.humancellatlas.org: https://schema.humancellatlas.org/a
+
+.. _metadata-schema: https://github.com/HumanCellAtlas/metadata-schema
+
+.. _schema-test-data: https://github.com/HumanCellAtlas/schema-test-data
+
+.. _dcp2: https://github.com/HumanCellAtlas/dcp2
+
+.. _DCP/2 system design: dcp2_system_design.rst
+
+
+Sizing schema changes
+---------------------
+
+A *schema change* is a set of related modifications to one or more of the 
+JSONSchema documents in the `metadata-schema`_ repository. That set must be
+chosen carefully. If a schema change *can* be made smaller by excluding a
+particular change to a schema document without sacrificing the desired end-user
+value, it *should* be made smaller, so as to not overwhelm the reviewers with a
+mix of orthogonal concerns. On the other hand, the set of changes shoudn't be
+too small either: all schema document changes that are necessary to achieve the
+desired end-user value should be combined in one overall schema change. If
+certain end user value is achieved in a series of many small schema changes,
+reviewers will likely not be able to see the big picture until they review the
+changes at the end of that series at which point it may be too late to reverse
+course.
+
+Some schema changes warrant an update to the `DCP/2 System Design`_
+specification. Whether they do or not is established during the review process
+of *pull requests* (PRs) against the `metadata-schema` and `schema-test-data`
+repositories. Specification updates are done as a PR against the `dcp2`_
+repository.
+
+
+Schema change authorship
+------------------------
+
+Anyone can author a PR. The PRs for one schema change don't all need to be by
+the same author, as long as there are no competing and open PRs in the same
+repository and for the same schema change. One person *should* author the PRs
+against the `metadata-schema`_ and `dcp2`_ repositories. That person is referred
+to as the *schema change author*. The schema change author can delegate
+authorship of PRs to other members of the DCP consortium.
+
+
+Schema change procedure
+-----------------------
+
+The overall procedure for authoring and reviewing a schema change consists of
+the following steps:
+
+1)  If it is considered unnecessary to update the specification and a
+    specification update wasn't explicitly requested, proceed to step 4.
+
+2)  Prepare a PR against the `dcp2`_ repository, updating the `DCP/2 System
+    Design`_.
+
+3)  Initiate a `Pull request review`_ of your PR. Once the PR has been approved
+    by the DCP, move on to the next step. Don't merge the PR just yet.
+
+4)  Prepare a PR against the ``develop`` branch of the `metadata-schema`_
```

Comment:
That sounds odd. Surely there must be a branch that corresponds to your development environment. This would rob us of the possibility to review multiple PRs in parallel.

Reply:
@mshadbolt we are still merging to develop before merging to staging

Basically, the overall flow now is `many related feature branches` --> `dev` --> `staging` 

And the review by the downstream components happens in the PR from staging to master

>That sounds odd. Surely there must be a branch that corresponds to your development environment. This would rob us of the possibility to review multiple PRs in parallel.

Even with the above in mind, we still don't have the possibility to have parallel PRs for unrelated updates. Changes need to be merged into an environment to be published and therefore can't be in the feature branches only. We can only generate valid test data if it's published.

Reply:
> Even with the above in mind, we still don't have the possibility to have parallel PRs for unrelated updates.

You *can* have multiple PRs open against `develop`. That's what this SOP assumes.

Reply:
Yep disregard the comment, I misread this for the step 6.